src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer started its engine and began to work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer turned off its engine and stopped working.");
```

Comment:
Similar to the doWork() method, the message in stopWork() should be more informative and indicate that it's a 'Bulldozer' that has stopped working.